.eslintrc.js

```diff
@@ -152,7 +156,6 @@ module.exports = {
     semi: "off",
   },
   settings: {
-    "import/internal-regex": "^@chainsafe/",
```

Comment:
why was this removed? should add some context in the PR

Reply:
First it was contextually not correct. Secondly it was making the mono-repo packages as external.